### DISCUSSION TITLE
Updates to Installation Guide - Common Issues Getting Started

### DISCUSSION THREAD

**Discussion by aimeewirick** [TreyWW/MyFinances#387] [Ideas]
I am a wannabe first time contributor, and noticed some issues when I was uploading MyFinances and I was wondering if I could contribute some updates to the 'Common issues/debugging' section of the installation guide to help others that might run into these?  Specifically, as a new user I ran into issues with installing on a POSIX platform, issues installing poetry, issues with some middleware installations that I was missing instructions on, including some django middleware,  forex_python, and mypy_boto3_iam as well as functions it uses like scheduler, events, and stepfunctions.  I have solutions for these and would love to contribute step-by-step instructions if that would be helpful?

**Comment by TreyWW** [comment]
Hey @aimeewirick, so sorry I missed your message. That would be incredibly helpful, youre welcome to create an issue / draft PR as you start making changes!

Thanks for the idea of more docs, thats one thing we're lacking at the moment lol
